elearning-frontend/src/components/admin/EditWords.js

```diff
@@ -0,0 +1,323 @@
+            category_id: '',
+            word: '',
+            choices: {
+                firstChoice: {
+                    choice: '',
+                    is_correct: false
+                },
+                secondChoice: {
+                    choice: '',
+                    is_correct: false
+                },
+                thirdChoice: {
+                    choice: '',
+                    is_correct: false
+                },
+                fourthChoice: {
+                    choice: '',
+                    is_correct: false
+                },
+            },
+            correct_answer: ''
+        })
+    }
+
+    // Set checkbox states
+    const setChoiceValue = (selectedIndex, input) => {
+        setWord({
+            ...word,
+            choices:
+                Object.entries(word.choices).map(([key, value], index) => {
+                    var currentIndex = index + 1
+                    return {
+                        choice: (currentIndex === selectedIndex) ? input : value.choice,
+                        is_correct: value.is_correct,
+                        label: value.label
+                    }
+                }).reduce(labelArray, {})
+        })
+    }
+
+    // Set checkbox states
+    const setCorrectAnswer = (selectedIndex, type, input) => {
+        setWord({
+            ...word,
+            choices:
+                Object.entries(word.choices).map(([key, value], index) => {
+                    var currentIndex = index + 1
+                    return {
+                        choice: value.choice,
+                        is_correct: (currentIndex === selectedIndex) ? !value.is_correct : false,
+                        label: value.label
+                    }
+                }).reduce(labelArray, {})
+        })
+    }
+
+    var view_element = ""
+
+    if (changeWordData) {
+        view_element = <tr><td colSpan={5} className='text-center'> LOADING DATA</td></tr>
+    }
+    else {
+        view_element =
+        wordList.map((word) => {
+            return <tr key={ word.id }>
+                <td className='text-center'>{ word.category.title }</td>
+                <td className='text-center'>{ word.word }</td>
+                <td className='text-center'>
+                    {
+                        Object.entries(word.choices).map(([key, value], index) => {
+                            return <p key={ key }
+                                        className={`text-center ${index % 2 == 0 ? 'bg-slate-200' : ''}`}>
+                                        { value.choice }
+                                    </p>
+                        })
+                    }
+                </td>
+                <td className='text-center'>{ word.correct_answer }</td>
+                <td className='flex'>
+                    <button onClick={(e) => {
+                        e.preventDefault()
+                        fillWord(word.id)
+                    }}>Edit</button>
```

Comment:
since you have already a function, transfer the preventDefault there so this line will be cleaner